core/server/api/invites.js

```diff
@@ -59,7 +59,7 @@ const invites = {
         return pipeline(tasks, options);
     },
 
-    destroy: (options) => {
+    destroy(options) {
```

Comment:
There are some more named functions 

> .then(function onModelResponse(model) {

Not sure this is rly helpful. If an error occurs in this fn, the stack would show the line number.
What do you think?